example/progress.js

```diff
@@ -1,17 +1,27 @@
+const readline = require('readline');
 const path = require('path');
 const fs   = require('fs');
 const ytdl = require('..');
 
-var url = 'https://www.youtube.com/watch?v=WhXefyLs-uw';
-var output = path.resolve(__dirname, 'video.mp4');
+const url = 'https://www.youtube.com/watch?v=WhXefyLs-uw';
+const output = path.resolve(__dirname, 'video.mp4');
 
-var video = ytdl(url);
+const video = ytdl(url);
+let starttime;
 video.pipe(fs.createWriteStream(output));
-video.on('progress', function(chunkLength, downloaded, total) {
-  process.stdout.cursorTo(0);
-  process.stdout.clearLine(1);
-  process.stdout.write((downloaded / total * 100).toFixed(2) + '% ');
+video.once('response', () => {
+  starttime = Date.now();
 });
-video.on('end', function() {
-  process.stdout.write('\n');
+video.on('progress', (chunkLength, downloaded, total) => {
+  const floatDownloaded = downloaded / total;
+  const downloadedMinutes = (Date.now() - starttime) / 1000 / 60;
+  readline.cursorTo(process.stdout, 0);
+  process.stdout.write(`${(floatDownloaded * 100).toFixed(2)}% downloaded`);
+  process.stdout.write(`(${(downloaded / 1024 / 1025).toFixed(2)}MB of ${(total / 1024 / 1024).toFixed(2)}MB)\n`);
```

Comment:
Was wondering why the total downloaded never equaled the total when at 100%. There's a `1025` here instead of `1024`